docs/guide/static-deploy.md

```diff
@@ -118,6 +118,14 @@ Now the `preview` command will launch the server at `http://localhost:8080`.
            uses: actions/deploy-pages@v2
    ```
 
+3. If you're deploying to GitHub Pages with a custom domain (eg. `www.example.com`), the following steps may need to be taken to avoid errors.
+
+   Set the `base` in `vite.config.js` to `'/'`.
+
+   Ensure that your `index.html` correctly points to the root file by adding a `.` in front of the path (eg. `./src/main.jsx`).
+
+   Set the `homepage` property in your `package.json` to `https://<USERNAME>.github.io/<REPO>/`.
```

Comment:
I don't think these are necessary. It should work without these changes.

Reply:
I ran an additional test to check this, and I agree that these changes are unnecessary if the base is correctly set. Will omit.